src/main/java/me/totalfreedom/totalfreedommod/shop/Shop.java

```diff
@@ -1,446 +1,446 @@
-
-        player.closeInventory();
-
-        player.sendMessage(plugin.sh.getShopPrefix() + " " + ChatColor.GREEN + "Successfully purchased the \"" + shopItem.getColoredName() + ChatColor.GREEN + "\" for " + ChatColor.GOLD + price + ChatColor.GREEN + "!");
-
-        if (shopItem.equals(ShopItem.GRAPPLING_HOOK))
-        {
-            player.sendMessage(ChatColor.GREEN + "Run /grapplinghook to get one!");
-        }
-        else if (shopItem.equals(ShopItem.LIGHTNING_ROD))
-        {
-            player.sendMessage(ChatColor.GREEN + "Run /lightningrod to get one!");
-        }
-        else if (shopItem.equals(ShopItem.FIRE_BALL))
-        {
-            player.sendMessage(ChatColor.GREEN + "Run /fireball to get one!");
-        }
-        else if (shopItem.equals(ShopItem.RIDEABLE_PEARL))
-        {
-            player.sendMessage(ChatColor.GREEN + "Run /rideablepearl to get one!");
-        }
-
-    }
-
-    public ShopItem getShopItem(int slot)
-    {
-        for (ShopItem shopItem : ShopItem.values())
-        {
-            if (shopItem.getSlot() == slot)
-            {
-                return shopItem;
-            }
-        }
-        return null;
-    }
-
-    @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
-    public void onPlayerQuit(PlayerQuitEvent event)
-    {
-        final String ip = Ips.getIp(event.getPlayer());
-        dataMap.remove(ip);
-    }
-
-    @EventHandler(priority = EventPriority.NORMAL)
-    public void onPlayerVote(VotifierEvent event)
-    {
-        Vote vote = event.getVote();
-        String name = vote.getUsername();
-        int coinsPerVote =  ConfigEntry.SHOP_COINS_PER_VOTE.getInteger();
-        Player player = server.getPlayer(name);
-        ShopData data = null;
-        if (player != null)
-        {
-            data = plugin.sh.getData(player);
-        }
-        else
-        {
-            data = plugin.sh.getData(name);
-        }
-
-        if (data != null)
-        {
-            data.setCoins(data.getCoins() + coinsPerVote);
-            data.setTotalVotes(data.getTotalVotes() + 1);
-            save(data);
-            FUtil.bcastMsg(ChatColor.GREEN + name + ChatColor.AQUA + " has voted for us on " + ChatColor.GREEN + vote.getServiceName() + ChatColor.AQUA + "!");
-        }
-
-        if (player != null)
-        {
-            player.sendMessage(ChatColor.GREEN + "Thank you for voting for us! Here are " + coinsPerVote + " coins!");
-        }
-    }
-
-    public Collection<ShopData> getLoadedData()
-    {
-        return dataMap.values();
-    }
-
-    protected File getConfigFile(String name)
-    {
-        return new File(getConfigFolder(), name.toLowerCase() + ".yml");
-    }
-
-    protected YamlConfig getConfig(ShopData data)
-    {
-        final YamlConfig config = new YamlConfig(plugin, getConfigFile(data.getUsername()), false);
-        config.load();
-        return config;
-    }
+package me.totalfreedom.totalfreedommod.shop;
```

Comment:
This entire file hasn't changed, and doing this is gonna break the Git history (OR make it difficult to work with)